src/haconiwa/scan/cli.py

```
@@ -0,0 +1,330 @@
+    path: Optional[Path] = typer.Option(None, "--path", "-p", help="Base path to search in"),
+    output: Optional[Path] = typer.Option(None, "--output", "-o", help="Output file path"),
+    format: str = typer.Option("table", "--format", "-f", help="Output format")
+):
+    """Compare multiple AI models"""
+    if len(models) < 2:
+        typer.echo("Error: At least 2 models required for comparison", err=True)
+        raise typer.Exit(1)
+    
+    comparator = ModelComparator(base_path=path or Path.cwd())
+    
+    results = comparator.compare_models(models)
```

Pylint (3.3.7)

[error] Instance of 'ModelComparator' has no 'compare_models' member

(E1101)